firewall/interception/windowskext/kext.go

```diff
@@ -25,108 +25,57 @@ var (
 
 	winErrInvalidData = uintptr(windows.ERROR_INVALID_DATA)
 
-	kext           *WinKext
 	kextLock       sync.RWMutex
 	ready          = abool.NewBool(false)
 	urgentRequests *int32
+	driverPath     string
+
+	kextHandle windows.Handle
 )
 
+const driverName = "PortmasterKext"
+
 func init() {
 	var urgentRequestsValue int32
 	urgentRequests = &urgentRequestsValue
 }
 
-// WinKext holds the DLL handle.
-type WinKext struct {
-	sync.RWMutex
-
-	dll        *windows.DLL
-	driverPath string
-
-	init               *windows.Proc
-	start              *windows.Proc
-	stop               *windows.Proc
-	recvVerdictRequest *windows.Proc
-	setVerdict         *windows.Proc
-	getPayload         *windows.Proc
-	clearCache         *windows.Proc
+// Init initializes the DLL and the Kext (Kernel Driver).
+func Init(path string) error {
+	driverPath = path
+	return nil
 }
 
-// Init initializes the DLL and the Kext (Kernel Driver).
-func Init(dllPath, driverPath string) error {
+// Start intercepting.
+func Start() error {
+	kextLock.Lock()
+	defer kextLock.Unlock()
 
-	new := &WinKext{
-		driverPath: driverPath,
-	}
+	filename := `\\.\` + driverName
 
+	// check if driver is already installed
 	var err error
-
-	// load dll
-	new.dll, err = windows.LoadDLL(dllPath)
-	if err != nil {
-		return err
+	kextHandle, err = openDriver(filename)
+	if err == nil {
+		return nil // device was already initialized
 	}
 
-	// load functions
-	new.init, err = new.dll.FindProc("PortmasterInit")
-	if err != nil {
-		return fmt.Errorf("could not find proc PortmasterStart in dll: %s", err)
-	}
-	new.start, err = new.dll.FindProc("PortmasterStart")
+	// initialize and start driver service
+	service, err := driverInstall(driverPath)
 	if err != nil {
-		return fmt.Errorf("could not find proc PortmasterStart in dll: %s", err)
+		return fmt.Errorf("Failed to start service: %s", err)
```

Comment:
Error messages should always start lower case.
Also, please use `%w` to wrap the error.